test/lib/tasks/message_queue_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class MessageQueueConsumerRakeTest < ActiveSupport::TestCase
+  context "when consuming messages from RabbitMQ" do
+    should "use GovukMessageQueueConsumer::Consumer" do
+      tagging_updater = mock('TaggingUpdater')
+      TaggingUpdater.expects(:new).returns(tagging_updater)
+
+      consumer = mock('consumer')
+      consumer.expects(:run).returns(true)
+
+      GovukMessageQueueConsumer::Consumer.expects(:new)
```

Comment:
Is it not possible to actually instantiate the `GovukMessageQueueConsumer::Consumer` and then expect it to receive `run`? Otherwise this test only duplicates what we've written in the Rakefile - when the consumer API changes this will still pass.